### PR TITLE
Improved field of view doc

### PR DESCRIPTION
**Pull request by DavidMansolino**
Linked to omichel/webots#3747

### CODE REVIEW

**Comment by DavidMansolino** [comment]
:+1:

File: reference/camera.md [hunk 1]
```diff
@@ -29,8 +29,7 @@ eye which is spherically distorted.
 
 ### Field Summary
 
-- `fieldOfView`: horizontal field of view angle of the camera. The value ranges
-from *0* to π radians. Since camera pixels are squares, the vertical field of
+- `fieldOfView`: horizontal field of view angle of the camera. The value is limited to the range *0* to π radians if the `spherical` field is set to FALSE, otherwise there is no upper limit. Since camera pixels are squares, the vertical field of
```
Comment (omichel) [inline reference/camera.md]: 0 and pi should be either both highlighted or none highlighted. Currently only 0 is highlighted.

**Comment by DavidMansolino** [comment]
:+1:

**Comment by omichel** [comment]
:+1:

**Pull request merged**